addons/mail/static/src/new/rtc/call_participant_card.js

```diff
@@ -92,12 +103,42 @@ export class CallParticipantCard extends Component {
         });
     }
 
-    onContextMenu() {
-        return; // TODO redirect click to volume menu anchor
-    }
-
-    onClickVolumeAnchor(ev) {
+    /**
+     * @param {Event} ev
+     */
+    onContextMenu(ev) {
+        ev.preventDefault();
         markEventHandled(ev, "CallParticipantCard.clickVolumeAnchor");
-        // TODO volume popover
+        if (this.closePopover) {
+            this.closePopover();
+            this.closePopover = undefined;
+            return;
+        }
+        if (!this.contextMenuAnchorRef?.el) {
+            return;
+        }
+        this.closePopover = this.popover.add(
+            this.contextMenuAnchorRef.el,
+            CallContextMenu,
+            {
+                onChangeVolume: (ev) => {
+                    const volume = Number(ev.target.value);
+                    this.userSettings.saveVolumeSetting({
+                        guestId: this.rtcSession?.guestId,
+                        partnerId: this.rtcSession?.partnerId,
```

Comment:
Again, `rtcSession` could reconcile `guestId` and `partnerId` to relate to persona.
It's the function very close to RPC that translate the persona to `partner_id` or `guest_id` for server.